src/rust/engine/remote_provider/remote_provider_opendal/src/lib.rs

```diff
@@ -298,7 +298,24 @@ impl ByteStoreProvider for Provider {
             }
 
             let path = self.path(digest.hash);
-            match self.operator.is_exist(&path).await {
+
+            if let Some(mut workunit_store_handle) = workunit_store::get_workunit_store_handle() {
+                workunit_store_handle
+                    .store
+                    .increment_counter(Metric::RemoteStoreExistsAttempts, 1);
+            }
```

Comment:
Thanks, yeah, I've put it in https://github.com/pantsbuild/pants/pull/20143 and will get that one in first, because there's a few other places that could benefit from this helper.